modify-publication/src/test/java/no/unit/nva/publication/modify/ModifyPublicationHandlerTest.java

```diff
@@ -122,8 +116,7 @@ public void handlerReturnsBadRequestResponseOnMissingPathParam() throws IOExcept
     @Test
     @DisplayName("handler Returns BadRequest Response On Missing Headers")
     public void handlerReturnsBadRequestResponseOnMissingHeaders() throws IOException {
-        Map<String, Object> event = Map.of(HEADERS, Collections.emptyMap());
-        InputStream inputStream = new ByteArrayInputStream(objectMapper.writeValueAsBytes(event));
+        InputStream inputStream = requestObjectToApiGatewayRequestInputSteam(null, null);
```

Comment:
Couldn't BAD_REQUEST be returned in a case of an empty object? Are we sure that the BAD_REQUEST will always be because of the empty headers and not the empty body?

Reply:
Good question. I will update the test.